lib/ansible/modules/cloud/azure/azure_rm_cosmosdbaccount_facts.py

```diff
@@ -0,0 +1,513 @@
+#!/usr/bin/python
+#
+# Copyright (c) 2019 Zim Kalinowski, (@zikalino)
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = '''
+---
+module: azure_rm_cosmosdbaccount_facts
+version_added: "2.8"
+short_description: Get Azure Cosmos DB Account facts.
+description:
+    - Get facts of Azure Cosmos DB Account.
+
+options:
+    resource_group:
+        description:
+            - Name of an Azure resource group.
+    name:
+        description:
+            - Cosmos DB database account name.
+    tags:
+        description:
+            - Limit results by providing a list of tags. Format tags as 'key' or 'key:value'.
+    retrieve_keys:
+        description:
+            - Retrieve keys and connection strings.
+        type: str
+        choices:
+            - all
+            - readonly
+    retrieve_connection_strings:
+        description:
+            - Retrieve connection strings.
+        type: bool
+
+extends_documentation_fragment:
+    - azure
+
+author:
+    - "Zim Kalinowski (@zikalino)"
+
+'''
+
+EXAMPLES = '''
+  - name: Get instance of Database Account
+    azure_rm_cosmosdbaccount_facts:
+      resource_group: testrg
```

Comment:
we should think over to be consistent on names of resources in docs and samples.

Reply:
yes, i am thinking of making updates for all our modules, this will be simple doc prs